kafka-ui-react-app/.eslintignore

```diff
@@ -1 +1,2 @@
 /src/generated-sources/**
+/src/setupTests.js/**
```

Comment:
It's not the best way to deal with eslint errors. Let's remove this line and disable rules inside of a file.